config/logging.yml

```diff
@@ -0,0 +1,15 @@
+# you can override this using by setting a system property, for example -Des.logger.level=DEBUG
+es.logger.level: INFO
+rootLogger: ${es.logger.level}, console
+logger:
+  # log action execution errors for easier debugging
+  action: DEBUG
```

Comment:
Let's keep it on `WARN`. If we see many errors we can always change it afterwards.